tolstoy/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "mentat_tolstoy"
+version = "0.0.1"
+workspace = ".."
+authors = ["Grisha Kruglov <gkruglov@mozilla.com>"]
+
+[dependencies]
+futures = "0.1"
+hyper = "0.11"
+tokio-core = "0.1"
+serde = "1.0"
+serde_json = "1.0"
+serde_derive = "1.0"
+lazy_static = "0.2"
+uuid = { version = "0.5", features = ["serde", "v4"] }
```

Comment:
We should make this `uuid` feature change in every Cargo file.

Reply:
Made this change in `edn/Cargo.toml` as well, but IIUC it's a preemptive one.

Reply:
I should really understand how feature stacking works in Cargo; what's the behavior with and without the change?

Reply:
We re-export Uuid from `edn` and use it across the codebase. I'm not sure what happens in case of feature... clashing, but it's best to either be consistent throughout or just centralize and re-use `edn::Uuid` here as well, to avoid confusion.